app/assets/i18n/strings_it.i18n.json

```diff
@@ -351,7 +351,7 @@
     }
   },
   "tray": {
-    "@info": "Apple Guidelines are very strict about the 'close' wording.",
+    "@info": "Le linee guida Apple sono molto rigide riguardo alla dicitura 'close'.",
```

Comment:
Comments (starting with `@`) don't need to be translated and are likely to be reverted